setup.py

```diff
@@ -103,6 +104,7 @@
     'raven[flask]~=6.0,>=6.2.1',
     'refextract~=0.0,>=0.2.2',
     'requests~=2.0,>=2.18.4',
+    'scikit-learn~=0.0,>=0.19.1',
```

Comment:
Strictly speaking these are not necessary, since `beard` already requires them. But the general principle I tried to follow in `setup.py` has always been "if it is explicitly imported, it must be part of `setup.py`".